internal/blueprint/ca_customizations.go

```diff
@@ -0,0 +1,5 @@
+package blueprint
+
+type CACustomization struct {
```

Comment:
I think this belongs in the customizations.go with the rest of the structs. Putting it into a separate file makes it harder to read the code.